crates/holochain/tests/tests/sharded_gossip/mod.rs

```diff
@@ -1,844 +0,0 @@
-        hashes.push(hash);
-    }
-
-    await_consistency(10, [&cells[0], &cells[1]]).await.unwrap();
-
-    println!(
-        "Done waiting for consistency between first two nodes. Elapsed: {:?}",
-        start.elapsed()
-    );
-
-    let records_0: Vec<Option<Record>> = conductors[0]
-        .call(&zomes[0], "read_multi", hashes.clone())
-        .await;
-    let records_1: Vec<Option<Record>> = conductors[1]
-        .call(&zomes[1], "read_multi", hashes.clone())
-        .await;
-    assert_eq!(
-        records_1.iter().filter(|r| r.is_some()).count(),
-        num,
-        "couldn't get records at positions: {:?}",
-        records_1
-            .iter()
-            .enumerate()
-            .filter_map(|(i, r)| r.is_none().then_some(i))
-            .collect::<Vec<_>>()
-    );
-    assert_eq!(records_0, records_1);
-
-    // Forget the first node's peer info before it gets gossiped to the third node.
-    // NOTE: this simulates "leave network", which we haven't yet implemented. The test will work without this,
-    // but there is a high chance of a 60 second timeout which flakily slows down this test beyond any acceptable duration.
-    conductors.forget_peer_info([cells[0].agent_pubkey()]).await;
-    conductors[0].shutdown().await;
-
-    // Bring a third conductor online
-    conductors.add_conductor_from_config(config).await;
-
-    // conductors.persist_dbs();
-
-    let (cell,) = conductors[2]
-        .setup_app("app", [&dna_file])
-        .await
-        .unwrap()
-        .into_tuple();
-    let zome = cell.zome(SweetInlineZomes::COORDINATOR);
-    SweetConductor::exchange_peer_info([&conductors[1], &conductors[2]]).await;
-
-    println!(
-        "Newcomer agent joined: agent={:#?}",
-        cell.agent_pubkey().to_kitsune()
-    );
-
-    conductors[2]
-        .require_initial_gossip_activity_for_cell(&cell, 2, Duration::from_secs(60))
-        .await
-        .unwrap();
-
-    println!(
-        "Initial gossip activity completed. Elapsed: {:?}",
-        start.elapsed()
-    );
-
-    await_consistency_advanced(
-        10,
-        (),
-        [(&cells[0], false), (&cells[1], true), (&cell, true)],
-    )
-    .await
-    .unwrap();
-
-    println!(
-        "Done waiting for consistency between last two nodes. Elapsed: {:?}",
-        start.elapsed()
-    );
-
-    let records_2: Vec<Option<Record>> = conductors[2].call(&zome, "read_multi", hashes).await;
-    assert_eq!(
-        records_2.iter().filter(|r| r.is_some()).count(),
-        num,
-        "couldn't get records at positions: {:?}",
-        records_2
-            .iter()
-            .enumerate()
-            .filter_map(|(i, r)| r.is_none().then_some(i))
-            .collect::<Vec<_>>()
-    );
-    assert_eq!(records_2, records_1);
-}
-
-#[cfg(feature = "test_utils")]
-#[tokio::test(flavor = "multi_thread")]
-async fn fullsync_sharded_local_gossip() -> anyhow::Result<()> {
```

Comment:
I don't see how this test makes sense. It's testing "local gossip", so two agents on the same conductor. There's no gossip involved, agents can just all read from one and the same DHT database.